src/main/scala/org/broadinstitute/clio/model/indexes.scala

```diff
@@ -1,18 +1,44 @@
 package org.broadinstitute.clio.model
 
+import java.time.OffsetDateTime
+
 case class ElasticsearchIndex(indexName: String, indexType: String, fields: Seq[ElasticsearchField])
 
 case class ElasticsearchField(fieldName: String, fieldType: Class[_])
 
 case class ReadGroup
 (
+  initiative: String,
```

Comment:
Whenever I see something not in some obvious order (lexicographic, numeric, whatevz),
I wonder what I'm missing.